src/kontrol/foundry.py

```diff
@@ -353,7 +353,7 @@ def help_info() -> list[str]:
             )
         res_lines.append('')
         res_lines.append(
-            'Access documentation for KEVM foundry integration at https://docs.runtimeverification.com/kevm-integration-for-foundry/'
+            'Access documentation for KEVM foundry integration at http://docs.runtimeverification.com/kontrol'
```

Comment:
I'd rather use `https` link:
```suggestion
            'Access documentation for KEVM foundry integration at https://docs.runtimeverification.com/kontrol'
```
I think there's auto-redirection from `http://docs.runtimeverification.com/kontrol` to `https://docs.runtimeverification.com/kontrol`, so it's mostly aesthetics.

Reply:
No worries, I could have noticed as well. I just changed to `https`.